src/components/Hamburger/Hamburger.tsx

```diff
@@ -131,15 +131,27 @@ export const Hamburger: FC<HamburgerProps> = ({
   labelActive = 'Close menu',
   labelInActive = 'Open menu',
   ...props
-}: HamburgerProps) => (
-  <Button
-    label={isActive ? labelActive : labelInActive}
-    variant="secondary"
-    size="kilo"
-    {...props}
-  >
-    <Box>
-      <Layers isActive={isActive} />
-    </Box>
-  </Button>
-);
+}: HamburgerProps) => {
+  const { label = undefined, component = 'hamburger', data = undefined } =
+    props.trackingPayload || {};
+
+  return (
+    <Button
+      label={isActive ? labelActive : labelInActive}
+      variant="secondary"
+      size="kilo"
+      {...props}
+      trackingPayload={{
```

Comment:
Would this become too much? After all, the path is already there, we could let the consumer decide what they wanna sent out for tracking.

Reply:
I'm not sure I understand your comment. "After all, the path is already there" — which path do you mean?

Reply:
I was thinking about should we set the value for default label here or not? Like what I demo here, we could add some default suffix string to the label or leave this to the consumer. For example in the marketing website, sometime we will not send GA event when close menu. So this suffix would become useless. 

Also, if we decide to add a suffix by default, we need to put it clearly in the documentation and maintain it properly. Would all the effort worth to do so?

Reply:
Ah, I get it now. Thanks for elaborating. I'd leave this to the consumer to decide, especially since you already a mention a use case where you need to modify it.